### PR TITLE
hey, a compressor! using default PCM ID. switchGears works too now

### PR DESCRIPTION
**Pull request by Team138Entropy**
but it might be backwards. more testing required.

### CODE REVIEW

**Review by maxh65000 — CHANGES_REQUESTED** [review]
This should be its own subsystem. We don't want to be adding stuff that might not exist on a testbed to our code as it might prevent us from testing it on testbeds.

**Review by maxh65000 — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -176,13 +187,16 @@ public void driveTrainLoop() {
       // Check User Inputs
       double DriveThrottle = mOperatorInterface.getDriveThrottle();
       double DriveTurn = mOperatorInterface.getDriveTurn();
+      boolean DriveShift = mOperatorInterface.getDriveShift();
       boolean AutoDrive = false;
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java:191]: Local variables should use camelCase. Please convert all of these User Input variables

File: src/main/java/frc/robot/subsystems/Drive.java [hunk 2]
```diff
@@ -181,7 +186,10 @@ public synchronized void setDrive(double throttle, double wheel, boolean quickTu
       Before gears are switched, it would be a good idea to check
       psi to ensure we can properly drive the piston
   */
-  public synchronized void SwitchGears(boolean HighGear) {}
+  public synchronized void SwitchGears() {
```
Comment (jhinspeter) [inline src/main/java/frc/robot/subsystems/Drive.java:189]: What's the use case here? Are we truly just going to have one button that toggles between low and high gears? Are there situations where we will want to ensure we are in one or the other?

**Review by jhinspeter — CHANGES_REQUESTED** [review]

**Comment by Team138Entropy** [comment]
Requested changes completed, see [#66](https://github.com/Team138Entropy/Entropy2020/pull/66)

**Pull request merged**